docker/package/Dockerfile-ubuntu-test

```diff
@@ -0,0 +1,12 @@
+# SPDX-FileCopyrightText: 2021 Oxhead Alpha
+# SPDX-License-Identifier: LicenseRef-MIT-OA
+
+FROM ubuntu:latest
```

Comment:
It will not suffice for us, since we want to be sure that everything works fine on every distribution we support (see `docker/supported_versions.json`. You can provide image tag via build arguments.

Reply:
Okay, but what version should we use on CI? I am asking, because if we set the version, then we would require to update it every time we move to the new version of os